src/tests/tests_gas_calibration.rs

```diff
@@ -83,11 +82,95 @@ fn test_basic_abi_call_counter() -> Result<()> {
     Ok(())
 }
 
+#[test]
+#[serial]
+fn test_basic_abi_call_counter_wasmv1() -> Result<()> {
+    let interface: Box<dyn Interface> = Box::new(TestInterface);
+    let module = include_bytes!(concat!(
+        env!("CARGO_MANIFEST_DIR"),
+        "/wasm/test_gc_abi_call_basic.wasm_add"
+    ));
+
+    let gas_costs = GasCosts::default();
+    // let runtime_module =
+    //     RuntimeModule::new(bytecode, 100_000, gas_costs.clone(),
+    // Compiler::SP)?;
+
+    let runtime_module =
+        RuntimeModule::new(module, 200_000, gas_costs.clone(), Compiler::SP)
```

Comment:
shouldn't we do the gas calibration with cranelift compiler for SC?

Reply:
If I remember correctly, we wanted to calibrate with SP, to get the "worst case gas", but maybe it has changed.

Reply:
> If I remember correctly, we wanted to calibrate with SP, to get the "worst case gas", but maybe it has changed.

I just don't know. Let's go for it for the moment.